third_party/python/riotwatcher/riotwatcher.py

```diff
@@ -244,6 +244,10 @@ def request_available(self):
 
 
 class RiotWatcher:
+
+    # Used for testing purpose, disabling https make all routes testable.
+    enable_https = True
```

Comment:
"Used for testing purpose < i.e. do not modify this for production code"
This i. e. is way clearer for me than the original sentence.